### DISCUSSION TITLE
Code Style

### DISCUSSION THREAD

**Discussion by marcus8448** [TeamGalacticraft/Galacticraft#190] [Meta]
We should probably come to agreement on what "clean" code looks like, for the tidiness/readability of the repo. We should **_probably_** (we don't have to) do this before 1.0.0 (So we don't have lots of stuff to go through).

**Comment by LemmaEOF** [comment]
If folks are ok with it, I'd prefer if we went with Fabric naming conventions: English word order (`GeneratorBlock` instead of `BlockGenerator`) and no `I`-prefixing interfaces.
